src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
 
-“He called me a juggahoe. You heard that before? I laughed in his face. I told him it was all family, but I’d beat his ass if that was how he needed to be educated.”
+::The Gutter - Golf cart
+{(if: $cartPartsSeen is 4)[I figured I'd done enough damage to the cart, so I decided to keep looking around. But man, [[fuck Ron’s shitty cart|The Gutter - Outside hub]].]}
+\(if: $cartPartsSeen is <4)[Ron drove everywhere in this beaten-to-hell golf cart. I had no idea how he'd gotten it in the first place, especially since he couldn't hold down a job longer than six months. But, he loved to roll around the tarps showing it off every time he added another dumbass accessory to the thing.
 
-“Too true, man.”
+He had almost everything “tricked” out, including:
+(if: $wheelsSeen is false)[>>[[the wheels|The Gutter - Golf cart - Wheels]]]
+(if: $defenseSeen is false)[>>[[the defense system|The Gutter - Golf cart - Defense]]]
+(if: $hornSeen is false)[>>[[the awful horn|The Gutter - Golf cart - Horn]]]
+(if: $engineSeen is false)[>>[[the engine|The Gutter - Golf cart - Engine]]]]
 
-At this, they both took a moment to nod solemnly in unison, as if the two of them could hear a song no one else could.
+(set: $golfCartSeen to true)
 
-[[>>I didn’t wanna intrude, so I turned back to the sign’s light by the entrance|The Gutter - Outside]]
-(if: $iHatchet is 1)[>> [[I tried to casually walk past them so I could hear more|The Gutter - Outside - old-school juggalos - part 2]]]
+::The Gutter - Golf cart - Wheels
+Old clown-painted Barbie and Ken heads were glued around the rims of each wheel. The face painting was... oddly detailed.
 
-::The Gutter - Outside - old-school juggalos - part 2 <754,644>
-“Yo little ‘lette!” The larger one called out. “That’s a sick fucking hatchet you got. Come over here!”
+(if: $multitool is true)[I used the multitool to unscrew a few crucial wheel bolts. Just a (link: "little goof")[(set: $cartDamage to true)(go-to: "The Gutter - Golf cart")].](else:)[[[back|The Gutter - Golf cart]]]
 
-[[I drew the hatchet and walked over|The Gutter - Outside - old-school juggalos - part 3]]
+(set: $cartPartsSeen to it +1)(set: $wheelsSeen to true)
 
-::The Gutter - Outside - old-school juggalos - part 3 <754,794>
-“Where’d you get that?” the shorter one asked.
+::The Gutter - Golf cart - Defense
+Ron had removed the entire protective top shell of the car for clearer 360° views, and he had duct-taped barbed wire around the frame in odd places.
 
-[[>>”Family heirloom.”|The Gutter - Outside - old-school juggalos - part 4a]]
-[[>>”Stole it off a couple juggahoes like you.”|The Gutter - Outside - old-school juggalos - part 4b]]
+(if: $multitool is true)[I used the blade of the multitool to cut the tape and carefully set the wire in a pile in the driver’s seat. I crossed my fingers for [[total ass destruction|The Gutter - Golf cart]].](else:)[[[back|The Gutter - Golf cart]]]
```

Comment:
I can't explain why, but in my heart, "Total Ass Destruction" ought to be capitalized.

Reply:
Far be it from me to deny what's in your heart.